bpo-39939: Add str.removeprefix and str.removesuffix (#18939)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2020-03-11-19-17-36.bpo-39939.NwCnAM.rst

> Added str.cutprefix and str.cutsuffix methods and corresponding bytes and bytearray methods to cut affixes off of a string, if present. Patch by Dennis Sweeney.

Author: ```suggestion
Added str.removeprefix and str.removesuffix methods and corresponding bytes and bytearray methods to remove affixes from a string if present. Patch by Dennis Sweeney.
```

> Added str.removeprefix and str.removesuffix methods and corresponding bytes and bytearray methods to remove affixes from a string if present. Patch by Dennis Sweeney.

Reviewer: You should mention the PEP 616 and collections.UserString.

> Added str.removeprefix and str.removesuffix methods and corresponding bytes, bytearray, and collections.UserString methods to remove affixes from a string if present. See PEP 616 for a full description. Patch by Dennis Sweeney.

Reviewer: FYI Sphinx syntax can be used here. For example, you can add a link to the PEP:

```suggestion
Added str.removeprefix and str.removesuffix methods and corresponding
bytes, bytearray, and collections.UserString methods to remove affixes
from a string if present.
See :pep:`616` for a full description.
Patch by Dennis Sweeney.
```

I also like to fit into 80 columns, but it's maybe just a personal preference :-D
